opentelemetry-sdk/src/logs/log_emitter.rs

```diff
@@ -128,10 +127,11 @@ impl LoggerProvider {
                 Err(LogError::Other(format!("{errs:?}").into()))
             }
         } else {
+            let error = LogError::Other("logger provider already shut down".into());
             otel_warn!(
-                name: "logger_provider_already_shutdown"
+                name: "LoggerProvider.Shutdown.AlreadyShutdown",
```

Comment:
this looks redundant, as returned Result already contains this information. If we need to keep this, then maybe keep it at debug/info level.

Reply:
So if the error is returned to the user, we decide not to log it even if this could be actionable ?

Reply:
not to log or log at a very low severity.

Do you see any reason why this should be logged, given Result already carries this information? "Internal logging" is more for things that are otherwise hard to figure out, right?

Reply:
I prefer logging at a lower severity (debug) for internal debugging. When we request logs from the customer at the debug level, they should include everything necessary for troubleshooting, even if the customer disregards the shutdown result.

Reply:
That is a good point! Agree with debug! here. It won't cause noise in normal operations, but when we ask users to report bug/issues, we can ask to collect all logs, including debug, so we'll have enough to investigate.

Reply:
Changed this to debug now.